spec/codebreaker/game_spec.rb

```diff
@@ -0,0 +1,227 @@
+require 'spec_helper'
+
+module Codebreaker
+  RSpec.describe Game do
+
+    SECRET_CODE_SIZE = 4
+
+    let(:game) { Game.new }
+
+    context '#guess' do
+      before { game.instance_variable_set(:@secret_arr, [1,2,3,4]) }
+
+      it { expect(game.guess('aaaaaa')).to eq false }
+      it { expect(game.guess('1234')).to eq '++++' }
+      it { expect(game.guess('4321')).to eq '----' }
+
+      it 'decreases attempts quantity' do
+        SECRET_CODE_SIZE.times { game.guess('1111') }
```

Comment:
expect { game.guess('1111') }. to change { game.instance_variable_get(:@attempts_quantity) }.from(n).to(n)